examples/database-seeding/arc.env.prod.yml

```diff
@@ -0,0 +1,14 @@
+services:
+  examples/database-seeding:latest:
+    debug:
+      path: .
+    ingress:
+      subdomain: api
+    parameters:
+      AUTO_DDL: none
+    dependencies:
+      my-demo-db:
+        interfaces:
+          postgres:
+            host: host.docker.internal
+            port: 5432
```

Comment:
I'm not sure this is a particularly useful file. I know it shows host-overrides, but the file cruft may distract from the specifics of this particular example. I'd recommend we only have 1 env file with a comment showing how they can change the AUTO_DDL value.